.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This is about project policy, so if @kc9wwh has no objection to adding this line, I'm fine with adding it.